ccloud/docs/beginner-cloud.rst

```diff
@@ -1,46 +1,43 @@
-
-.. _beginner-cloud:
+.. _ccloud-cli-tutorial:
 
 Tutorial: |ccloud| CLI
 =======================
 
 Overview
 --------
 
-This tutorial will show you how to use the `Confluent Cloud CLI
+This tutorial shows you how to use the `Confluent Cloud CLI
```

Comment:
How about we take the line from below and put it here like so: 

This tutorial shows you how to use the `Confluent Cloud CLI <https://docs.confluent.io/current/cloud/cli/install.html>`__ to interact with
your `Confluent Cloud <https://confluent.cloud/login>`__ cluster. The tutorial uses real resources in |ccloud|, and it creates and deletes topics, service accounts, API keys, and ACLs.

Reply:
Ack and done